### PR TITLE
Enable RBAC in graphql support

### PR DESCRIPTION
**Pull request by bzwei**

### CODE REVIEW

File: spec/controllers/v1.0/graphql_controller_spec.rb [hunk 1]
```diff
@@ -7,21 +8,47 @@
   let(:template_id) { template.id }
   let!(:workflows) { create_list(:workflow, 4, :template_id => template.id) }
   let(:id) { workflows.first.id }
+  let(:roles_obj) { double }
```
Comment (mkanoor) [inline spec/controllers/v1.0/graphql_controller_spec.rb]: @bzwei I think this should be an instance double
instance_double(RBAC::Role)

File: app/controllers/api/v1x0/actions_controller.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@ class ActionsController < ApplicationController
       def index
         stage = Stage.find(params.require(:stage_id))
 
-        RBAC::Access.enabled? ? collection(rbac_scope(stage.actions)) : collection(stage.actions)
+        collection(rbac_scope(stage.actions))
```
Comment (hsong-rh) [inline app/controllers/api/v1x0/actions_controller.rb]: Need to add `index_access_check` in `rbac_scope`

File: app/controllers/api/v1x0/requests_controller.rb [hunk 3]
```diff
@@ -39,6 +39,8 @@ def request_params
       end
 
       def rbac_scope(relation)
+        index_access_check
+
```
Comment (hsong-rh) [inline app/controllers/api/v1x0/requests_controller.rb]: Above `RBAC::Access.enabled? ? collection(rbac_scope(reqs)) : collection(reqs)` can simplify to
`collection(rbac_scope(reqs))`, as you did in `actions_controller.rb`

File: app/controllers/api/v1x0/stages_controller.rb [hunk 4]
```diff
@@ -20,6 +20,8 @@ def index
       private
 
       def rbac_scope(relation)
```
Comment (hsong-rh) [inline app/controllers/api/v1x0/stages_controller.rb]: Same comment as above.

**Pull request merged**